.travis.yml

```diff
@@ -3,34 +3,26 @@ services:
   - docker
 
 cache:
-  directories:
-    - $HOME/bootstrap-vz
   pip: {}
 
 addons:
   apt:
     packages:
-      - kpartx
-      - parted
-      - debootstrap
-      - python-termcolor
-      - python-jsonschema
-      - python-docopt
-      - python-yaml
+      - wget
+      - unzip
       - python-pip
+      - qemu-system-x86
```

Comment:
if we're using docker this shouldn't be necessary

if it is necessary, find out why, and remove it

Reply:
the Makefile looks for qemu-system-x86 on all targets